flax/core/scope.py

```diff
@@ -34,12 +34,13 @@
 PRNGKey = Any
 Array = Any
 
+Filter = Union[bool, str, Sequence[str]]
+CollectionFilter = Filter
+RNGFilter = Filter
```

Comment:
Have you considered naming this `RNGCollectionFilter`?

Reply:
I don't think collection is a good name here (it's a prng sequence not really a collection). I have considered `PRNGSequenceFilter`, It's a bit long though.

Reply:
Ah oops yes I meant `PRNGSequenceFilter`. OK I trust your judgement